### PR TITLE
add general support for moonriver

### PR DESCRIPTION
**Pull request by eshaben**
This PR includes the addition of general support for Moonriver. It does not update tutorials from Moonbase Alpha to Moonriver. It adds Moonriver configurations where needed.

This relies on the Moonriver network page going into `master` first

### CODE REVIEW

**Comment by eshaben** [comment]
blocked by #113

**Comment by eshaben** [comment]
I keep switching this back and forth from draft to ready...and I am doing it again haha. In reviewing this myself, I realized we should definitely add a MetaMask page for Moonriver so I'll add that now into this PR and it should be ready to be reviewed tomorrow. I'll make a note here when it's ready again.

**Comment by albertov19** [comment]
I agree, we will also need to do a video (@themacexpert) on how to connect MetaMask to Moonriver. Both the easy way (with the button) and the hard way (manually)

**Comment by eshaben** [comment]
ok there is a page now for connecting to MetaMask. Once the 1-click connect button is there for Moonriver, it can be added to this page. I'll try to push forward with the MetaMask button. 

This is ready for review 😅

**Comment by eshaben** [comment]
Updated Moonriver info for phase 1 and the governance, staking, treasury, etc. tables for consistency... this is good to be reviewed!

File: networks/moonriver.md [hunk 1]
```diff
@@ -15,10 +15,10 @@ In order to collect as much feedback as possible and provide fast issue resoluti
 
 ## Initial Configurations
 
-Moonriver is scheduled to follow a [5-phase launch process](https://moonbeam.network/networks/moonriver/launch/). Currently, Moonriver is in Phase 0 of the launch process and has the following configurations:
```
Comment (albertov19) [inline networks/moonriver.md:18]: what about creating a Phase Number variable so you can change it in 1 place :)

File: node-operators/networks/collator.md [hunk 2]
```diff
@@ -142,12 +156,15 @@ Make sure you write down this public key of the author ID. Next, this will be ma
...
+There is a bond that is sent when mapping your author ID with your account. This bond is per author ID registered. The bond set is as follows:
+
+ - Moonbase Alpha - {{ networks.moonbase.staking.collator_map_bond }} DEV tokens 
+ - Moonriver - {{ networks.moonriver.staking.collator_map_bond }} MOVR tokens. 
 
 The `authorMapping` module has the following extrinsics programmed:
 
- - **addAssociation**(*address* authorID) — maps your author ID to the H160 account from which the transaction is being sent, ensuring is the true owner of its private keys. It requires a {{ networks.moonbase.staking.collator_map_bond }} DEV tokens bond
- - **clearAssociation**(*address* authorID) — clears the association of an author ID to the H160 account from which the transaction is being sent, which needs to be the owner of that author ID. Also refunds the {{ networks.moonbase.staking.collator_map_bond }} DEV tokens bond
+ - **addAssociation**(*address* authorID) — maps your author ID to the H160 account from which the transaction is being sent, ensuring is the true owner of its private keys. It requires a bond
```
Comment (albertov19) [inline node-operators/networks/collator.md:166]: maybe add a hyperlink to where to find it?

File: node-operators/networks/collator.md [hunk 3]
```diff
@@ -34,9 +34,19 @@ From a technical perspective, collators must meet the following requirements:
...
+Similar to Polkadot validators, you need to create an account. For Moonbeam, this is an H160 account or basically an Ethereum style account from which you hold the private keys. In addition, you will need a minimum amount of tokens staked to be considered eligible (become a candidate). Only a certain amount of the top collators by nominated stake will be in the active set.
 
-Collators need to have a minimum of {{ networks.moonbase.staking.collator_min_stake }} DEV to be considered eligible (become a candidate). Only the top {{ networks.moonbase.staking.max_collators }} collators by nominated stake will be in the active set.   
+=== "Moonbase Alpha"
+    |    Variable     |                          Value                          |
+    |:---------------:|:-------------------------------------------------------:|
+    |  Minimum stake  | {{ networks.moonbase.staking.collator_min_stake }} DEV  |
```
Comment (albertov19) [inline node-operators/networks/collator.md:42]: Might be better to put, Bond amount? Instead of minimum stake

**Review by albertov19 — CHANGES_REQUESTED** [review]
Looks good 🚀 ! I've added some comments

File: networks/moonriver.md (hunk 1 above)
Comment (eshaben) [reply networks/moonriver.md:18]: I don't think it's worth it to put this one thing in a variable, as we do not reuse it, and I know to keep track of it

File: node-operators/networks/collator.md (hunk 2 above)
Comment (eshaben) [reply node-operators/networks/collator.md:166]: a hyperlink to find what?

Comment (albertov19) [reply node-operators/networks/collator.md:166]: You removed the bond information, you can add a link to where the table is with the bond information, like 'it requires a [bond](#link-to-section)`

**Review by albertov19 — APPROVED** [review]

**Comment by albertov19** [comment]
LGTM 🚀 🚢

**Pull request merged**